applications/mlflow/kots/k8s-app.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: "mlflow"
+spec:
+  descriptor:
+    version: "0.1.1"
+    description: "MLflow"
+    icon: https://mlflow.org/img/mlflow-black.svg
+    links:
+      - description: 🔗 Open MLflow
+        url: 'http://{{repl ConfigOption "mlflow_ingress_host" }}'
```

Comment:
^ I was also curious how the different urls in the k8s and kots app custom resources works. (Asking because it looks like something similar recently came up in a support issue that Craig worked on and I've been trying to figure out how to best capture this in the docs!)

Since the urls in the `ports.applicationUrl` and `descriptor.links.url` fields don't "match", how does that affect what KOTS does (or does not do) to rewrite these URLs/append the local port for embedded versus existing installs?

Reply:
Yeah, this is confusing. I think the discrepancy comes from the fact that `descriptor.links.url` support the `repl` templates but `ports.applicationUrl` doesn't - https://docs.replicated.com/reference/custom-resource-application#ports